src/fheroes2/gui/ui_button.h

```diff
@@ -293,6 +293,11 @@ namespace fheroes2
 
         void drawShadows( Image & output = Display::instance() );
 
+        int32_t getButtonsCount()
```

Comment:
We can make this method `const`:
```suggestion
        int32_t getButtonsCount() const
```